python_ta/typecheck/type_store.py

```diff
@@ -36,7 +37,15 @@ def __init__(self, type_constraints):
                     if gen == 'Generic':
                         self.classes[class_def.name]['__pyta_tvars'] = tvars
             for function_def in class_def.nodes_of_class(astroid.FunctionDef):
+                poly_tvars = []
                 arg_types = []
+                try:
```

Comment:
You don't need a try-except here. You can use the `dict` method `get` to avoid a `KeyError`.